components/public-api/gitpod/experimental/v1/workspaces.proto

```diff
@@ -27,6 +27,8 @@ service WorkspacesService {
     //   NOT_FOUND:           the workspace_id is unkown
     //   FAILED_PRECONDITION: if there's no running instance
     rpc StopWorkspace(StopWorkspaceRequest) returns (stream StopWorkspaceResponse) {}
+
+    rpc ControlPort(ControlPortRequest) returns (ControlPortResponse) {}
```

Comment:
Maybe we can call it `CreatePortPolicy` to remain closer to CRUD

Reply:
We actually considered to call it `UpdatePort`.